src/components/Parallax.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+// nodejs library that concatenates classes
+import classNames from "classnames";
+// nodejs library to set properties for components
+import PropTypes from "prop-types";
+// @material-ui/core components
+import { makeStyles } from "@material-ui/core/styles";
+
+// core components
+import styles from "../assets/jss/material-kit-react/components/parallaxStyle.js";
+
+const useStyles = makeStyles(styles);
+
+export default function Parallax(props) {
+  let windowScrollTop;
+  if (window.innerWidth >= 768) {
+    windowScrollTop = window.pageYOffset / 3;
+  } else {
+    windowScrollTop = 0;
+  }
+  const [transform, setTransform] = React.useState(
+    "translate3d(0," + windowScrollTop + "px,0)"
+  );
+  React.useEffect(() => {
```

Comment:
This is not how we usually use `useEffect`. we import it directly. Is this code copied and pasted or re-used?

Reply:
Yes, it's extracted from https://material-ui.com/store/items/material-kit/
It's adjusted but can be further improved